examples/data/mysql/src/main/java/io/helidon/examples/data/mysql/PokemonService.java

```diff
@@ -0,0 +1,113 @@
+
+class PokemonService implements HttpService {
+
+    // Helidon Data repository interface providing Pokemon entity operations.
+    private final PokemonRepository pokemonRepository = Services.get(PokemonRepository.class);
+    // Helidon Data repository interface providing Pokemon entity operations.
+    private final TypeRepository typeRepository = Services.get(TypeRepository.class);
+
+    // Pokemon endpoint routing rules
+    @Override
+    public void routing(HttpRules rules) {
+        rules.get("/all", this::all)
+                .get("/type/{name}", this::type)
+                .get("/get/{name}", this::pokemon)
+                .post("/", Handler.create(PokemonDto.class, this::insert))
+                .delete("/{id}", this::delete);
+    }
+
+    /**
+     * Handles the {@code GET /all} request and returns a list of all {@link Pokemon} entities.
+     *
+     * @param request  the server request
+     * @param response the server response
+     */
+    private void all(ServerRequest request, ServerResponse response) {
+        response.send(pokemonRepository.listOrderByName());
+    }
+
+    /**
+     * Handles the {@code GET /type/{name}} request and returns a list of {@link Pokemon} entities
+     * associated with a specific breed name.
+     *
+     * @param request  the server request
+     * @param response the server response
+     */
+    private void type(ServerRequest request, ServerResponse response) {
+        String name = request.path().pathParameters().get("name");
+        response.send(pokemonRepository.listByType_Name(name));
+    }
+
+    /**
+     * Handles the {@code GET /get/{name}} request and returns a {@link Pokemon} entity by its name.
+     *
+     * @param request  the server request
+     * @param response the server response
+     */
+    private void pokemon(ServerRequest request, ServerResponse response) {
+        String name = request.path().pathParameters().get("name");
+        response.send(pokemonRepository.findByName(name));
+    }
+
+    /**
+     * Handles the {@code POST /} request and inserts a new {@link Pokemon} entity.
+     * <p>
+     * Pokémon entity content is supplied as JSON object and translated to {@link PokemonDto} class by HTTP
+     * request processing layer.
+     * Pokémon type record must already exist. Pokémon entity must not exist in the database.
+     *
+     * @param pokemonDto the pokémon to insert into the database
+     * @param response   the server response
+     */
+    private void insert(PokemonDto pokemonDto, ServerResponse response) {
+        // Pokemon data modification task is executed as single database transaction
+        response.send(Tx.transaction(() -> {
+            // Retrieve Type entity for provided name
+            Type type = typeRepository.getByName(pokemonDto.type());
+            // Prepare and insert new Pokemon entity into the database
+            Pokemon pokemonEntity = new Pokemon(pokemonDto.name(), type);
```

Comment:
Since this is a Helidon SE example, can you use JSON-P instead of JSON-B and remove the JSON DTO.

See https://github.com/helidon-io/helidon-examples/blob/229eb0aade9b5adffae6feecb5628d0a9e5a466e/examples/dbclient/mysql/src/main/java/io/helidon/examples/dbclient/mysql/PokemonService.java#L85